.vscode-config/tasks.json

```diff
@@ -7,4 +7,4 @@
             "command": "cd Debug && rm -rf * && cmake -DCMAKE_BUILD_TYPE=Debug -DUNIT_TESTING=ON -DUNIT_TESTING_MOCK_ENABLED=ON .. && make"
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
lauch and tasks json for vscode could be removed.
otherwise, we need to update them to correct values

Reply:
> otherwise, we need to update them to correct values

What do you mean by correct values? File content or file name/path? These are intentionally written as they are, by Dane.